api/nodes/serializers.py

```diff
@@ -110,6 +118,7 @@ def get_pointers_count(self, obj):
         return len(obj.nodes_pointer)
 
     def create(self, validated_data):
+        validated_data.pop('type')
```

Comment:
You might be able to override `Serializer.is_valid` to remove these fields from the validated data, though I'm not sure if this is the best place to do that; it's worth investigating.